### DISCUSSION TITLE
subfinder: command not found

### DISCUSSION THREAD

**Discussion by sakibislam4iu** [projectdiscovery/subfinder#861] [Q&A] [answered]
subfinder: command not found show problem linux

**Comment by MetzinAround** [comment]
Hi @sakibislam4iu. Could you tell me more about your error? 

How did you install subfinder? We recommend using [PDTM](https://github.com/projectdiscovery/pdtm) to install and manage ProjectDiscovery tools.

If it can't be found, it's also likely a PATH issue. Here's a video that should help. 

https://youtu.be/wtHcXA2l9c4

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.

**Comment by jithesh82** [comment, marked as answer]
I faced the same problem.  The video link of @MetzinAround helped me.  I did 

`go env | grep GOPATH`

which gave GOPATH='/root/go'

So in my ~/.zshrc I exported the executable directory path of bin

`export PATH=$PATH:/root/go/bin`

And then sourced it

`source ~/.zshrc`

Now it works like a charm!

**Reply by dogancanbakir** [reply]
Thanks for sharing @jithesh82! And shoutout to @MetzinAround for the great video!

**Reply by MetzinAround** [reply]
Still helping out after all these years! 😸
